src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  --active-color--: #00acdc;
+}
+
+* {
+  border: 0;
+}
```

Comment:
CHECKLIST p.9. (I know it says about margins and paddings, but really you should use `*` only to set `box-sizing: border-box`)